app/code/Magento/Eav/Block/Adminhtml/Attribute/Edit/Js.php

```diff
@@ -36,12 +36,21 @@ public function __construct(
         parent::__construct($context, $data);
     }
 
+    /**
+     * @deprecated Misspelled method
+     * @see getCompatibleInputTypes
```

Comment:
Same question as https://github.com/magento/magento2/pull/13301/files#r163084396

Reply:
I was pointed out in [previous pr](https://github.com/magento/magento2/pull/13187#discussion_r161676584) to use that approach and it's working great in IDEs, using phpstorm my self.